common/flatpak-dir.c

```diff
@@ -7604,6 +7604,15 @@ export_desktop_file (const char         *app,
       /* Add a marker so consumers can easily find out that this launches a sandbox */
       g_key_file_set_string (keyfile, G_KEY_FILE_DESKTOP_GROUP, "X-Flatpak", app);
 
+      /* Disable krunner dbusplugins by default, so that flatpak applications cannot
+       * unintentionally grab sensitive search data.
+       */
+      if (g_key_file_get_boolean (keyfile, G_KEY_FILE_DESKTOP_GROUP,
```

Comment:
I don't quite understand what this means: If this key is unset, the default comes from KRunner itself. And just like the KPluginMetaData API, the default value is false.

Reply:
Installing a flatpak should do nothing by default. The user is expected to enable search providers manually. This is the behavior in GNOME.

Reply:
Exactly. @alex1701c, if we don't do this, a search provider could be active by default. That's a policy that flatpak does not like here. You've to go to the krunner settings and say "yes, I want this to be active".

Reply:
Oh, makes sense. Maybe mention that they should be enabled in the "Plasmasearch" systemsettings module.

We probably should document this somewhere on the KDE side or when individual plugins make use of this.